src/LiveComponent/src/Hydration/ArrayCollectionHydrationExtension.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LiveComponent\Hydration;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @author Jean-Paul van der Wegen <info@jpvdw.nl>
+ *
+ * @experimental
+ *
+ * @internal
+ */
+class ArrayCollectionHydrationExtension extends AbstractDoctrineHydrationExtension implements HydrationExtensionInterface
```

Comment:
DoctrineArrayCollection... would be more explicit i think

Reply:
Done